docs/source/conf.py

```diff
@@ -223,6 +223,8 @@
 # We've decided to rely on RTD's default robots.txt, so we no longer need to specify a custom one,
 # see discussion at https://github.com/kedro-org/kedro/issues/3741
 # html_extra_path = [str(here / "robots.txt")]
+html_extra_path = [str(here / "sitemap.xml")]
```

Comment:
👀 So this was it...

Reply:
Unfortunately, I checked the build, and it didn’t help. Something is still overwriting the `sitemap.xml`, even with that line included.